x-pack/plugins/security_solution/public/common/components/sourcerer/use_get_sourcerer_data_view.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMemo } from 'react';
+import { DataView } from '@kbn/data-views-plugin/public';
+import { useSourcererDataView } from '../../containers/sourcerer';
+import { useKibana } from '../../lib/kibana';
+import type { SourcererScopeName } from '../../store/sourcerer/model';
+
+/*
+ *
+ * returns the created dataView based on sourcererDataView spec
+ * returned from useSourcererDataView
+ *
+ * */
+export const useGetScopedSourcererDataView = ({
```

Comment:
We appear not to be using this hook anywhere... Is this PR still a WIP or is this not needed?